index.html

```diff
@@ -0,0 +1,553 @@
+                <p class="flex-three-p">Lorem ipsum dolor sit amet,
+                    consectetur adipiscingelit, do
+                    elusm tempor incididunt.
+                </p>
+            </div>
+        </div>
+        <div>
+            <i class="fa fa-usd flex-three-icon"></i>
+            <div>
+                <p>
+                    <div class="txt-underline">Donate&nbsp;</div>
+                </p><br>
+                <p class="flex-three-p">Lorem ipsum dolor sit amet,
+                    consectetur adipiscingelit, do
+                    elusm tempor incididunt.
+                </p>
+            </div>
+        </div>
+        <div>
+            <i class="fa fa-credit-card flex-three-icon"></i>
+            <div>
+                <p>
+                    <div class="txt-underline">Contribu</div>te
+                </p>
+                <p class="flex-three-p">Lorem ipsum dolor sit amet,
+                    consectetur adipiscingelit, do
+                    elusm tempor incididunt.
+                </p>
+            </div>
+        </div>
+        <div>
+            <div>
+                <i class="fa fa-user-plus flex-three-icon"></i>
+            </div>
+            <div>
+                <p>
+                    <div class="txt-underline">Participa</div>te
+                </p>
+                <p class="flex-three-p">Lorem ipsum dolor sit amet,
+                    consectetur adipiscingelit, do
+                    elusm tempor incididunt.
+                </p>
+            </div>
+        </div>
+        <div>
+            <i class="fa fa-wrench flex-three-icon"></i>
+            <div>
+                <p>
+                    <div class="txt-underline">Lnteract</div>ive Elements
+                </p>
+                <p class="flex-three-p">Lorem ipsum dolor sit amet,
+                    consectetur adipiscingelit, do
+                    elusm tempor incididunt.
+                </p>
+            </div>
+        </div>
+
+    </section>
+    </div>
+<!--================================================================================
+    scetion four ABOUT US
+====================================================================================-->
+    <section class="about-us">
+        <div class="common-container">
+            <h1> ABOUT US</h1>
+            <div class="drop-symbol">
+                <div class="line2"></div>
+                <div class="tint">
+                    <i class="fa fa-tint" aria-hidden="true"></i>
+                </div>
+                <div class="line3"></div>
+            </div><br><br>
+            <p class="ourclass-p">Charixy is the charity,Non-profit government organization,fundrasin. Our
+                activities help entreprenurs get the target.
+            </p>
+        </div>
+        <section class="flex-common flex-center">
+            <div>
+                <div>
+                    <h3>
+                        <div class="txt-underline">DONAT</div>ION
```

Comment:
If you break the text, then it becomes difficult for search engines to index this word.  Instead of indexing 'Donation' it will now get indexed as 'donat' and 'ion'.    There is no need for a div tag here. 

```
h3.txt-underline {
  position: relative;
}
h3:after {
  content: '';
  position: absolute;
  width: 25px;
  display: block;
  height: 5px;
  background-color: red;
  bottom: -5px;
}
```